dist/tools/suit/gen_key.py

```diff
@@ -37,7 +38,7 @@ def main():
                            encryption_algorithm=crypt,
                            )
 
-    with open(sys.argv[1], "wb") as f:
+    with open(os.open(sys.argv[1], os.O_CREAT | os.O_WRONLY, 0o600), "wb") as f:
```

Comment:
> If O_CREAT and O_EXCL are set, open() shall fail if the file exists. 

Says the linux man page of open. I assume that will also be true for the python wrapper then. Can we have this to avoid overwriting the key?